scripts/run-live-build.sh

```diff
@@ -93,14 +93,25 @@ if [[ "$RUN_TYPE" == "$ALL_RUN_TYPE" || "$RUN_TYPE" == "$VM_RUN_TYPE" ]]; then
 	cp -r "$TOP/live-build/config/hooks/$VM_RUN_TYPE/." "$build_dir/config/hooks"
 fi
 
+sed "s/@@PLATFORM@@/$APPLIANCE_PLATFORM/" \
+	<"$build_dir/config/package-lists/delphix-platform.list.chroot.in" \
+	>"$build_dir/config/package-lists/delphix-platform.list.chroot"
+
+if [[ -d "$TOP/live-build/variants/$APPLIANCE_VARIANT/package-lists" ]]; then
+	for list in "$TOP/live-build/variants/$APPLIANCE_VARIANT/package-lists/"*; do
+		[[ -f $list ]] || continue
```

Comment:
Should we fail instead, and perhaps check that the file name ends with `.chroot`?

Reply:
I don't see why we should.

Reply:
I believe the package would not be consumed unless it ends with `.chroot` or `.binary`, as the suffix tells live-build at which stage to consume that package-list. We currently do not have good reasons to install it during the `.binary` stage, which is why restricting it to `.chroot` makes the most sense.

Regarding the other check, there are no valid reasons to have sub-directories in here since they won't be used, so why just silently skip them instead of issuing a failure?

Reply:
The existing code that deals with package lists that apply to all variants does not enforce that package lists apply to any particular stage or that they end with `.chroot` or `.binary`. In fact, an existing `delphix-platform.list.chroot.in` file is currently passed to `live-build(7)`, which ignores it. This preëxisting issue could be fixed as you describe, but such a fix is orthogonal to this PR. This PR doesn't make the preëxisting issue any worse.

Regarding the other check, it is necessary to ensure the glob match was successful.

Reply:
You are right that we don't do the check for the existing list, although we probably should. If you want to improve the checking here while you are already modifying this logic, that would be welcome, but if not feel free to drop the issue.